runtimes/ruby/ruby-on-rails.md

```diff
@@ -69,15 +68,7 @@ releases, but, hey, we're human!
 Due to current landscape in ruby applications, the default version is the
 greatest 2.3.Y. We provide also the 2.2.Y and 2.1.Y versions.
 
-
-## Environment injection
-
-To access your variable in your application, nothing simpler! Just get
-it from your environment, like you would with `PATH`:
-`ENV["MY_VARIABLE"]`.
-
-
-## Configuration secret key production
+### Configuration secret key production
```

Comment:
Why the title change here ? Like the "choose ruby version", I like the fact that it's in the summary